README.md

```diff
@@ -1,33 +1,140 @@
-# Storyblocks-api [![Build Status](https://secure.travis-ci.org/noblesamurai/storyblocks-api.png?branch=master)](http://travis-ci.org/noblesamurai/storyblocks-api) [![NPM version](https://badge-me.herokuapp.com/api/npm/storyblocks-api.png)](http://badges.enytc.com/for/npm/storyblocks-api)
+# Storyblocks API [![Build Status](https://travis-ci.org/noblesamurai/node-storyblocks-api.svg?branch=master)](https://travis-ci.org/noblesamurai/node-storyblocks-api)
 
 > storyblocks API wrapper for node.js
 
-## Purpose
-- What problem does this module solve? At least a few sentences.
-PLEASE_FILL_IN_HERE
+## Installation
 
-## Usage
+This module is installed via npm:
 
-```js
-// Several examples of usage.
-// Usually copying and pasting code from the tests and making the code standalone suffices.
-// PLEASE_FILL_IN_HERE
+``` bash
+$ npm install storyblocks-api
 ```
 
 ## API
 
-PLEASE_FILL_IN_HERE
+Create the authorised API service you need (audioblocks or videoblocks).
 
-Note: To regenerate this section from the jsdoc run `npm run docs` and paste
-the output above.
+```js
+const credentials = {
+  privateKey: '...',
+  publicKey: '...'
+};
+const { videoblocks } = require('storyblocks-api')(credentials);
+```
 
-## Installation
+Then start making your requests to storyblocks
+```js
+const params = {
+  keywords: 'mountain aerial',
+  page: 1,
+  numResults: 5
+};
+const results = await videoblocks.search(params);
```

Comment:
We don't really say exactly what `results` is.  I assume it just the javascript object version of the storyblocks JSON response?  Might be worth saying so.